client/blocks/comment-detail/comment-detail-header.jsx

```diff
@@ -82,7 +82,9 @@ export const CommentDetailHeader = ( {
 						</span>
 					</div>
 					<div className="comment-detail__author-info-element">
-						{ translate( 'on %(postTitle)s', { args: { postTitle } } ) }
+						{ translate( 'on %(postTitle)s', { args: {
+							postTitle: postTitle ? decodeEntities( postTitle ) : translate( 'Untitled' ),
```

Comment:
Sure, I think that'd work. I was mainly thinking about my image posts that have no title and no text content though. "Untitled" isn't the best fallback in that scenario but might be fine for now. If we can, I proposed we fallback to the post content, and if no text content, then fallback to "Untitled".

Might be nice if it displayed what was in the post content, in text form, like `[image]` or something like that.